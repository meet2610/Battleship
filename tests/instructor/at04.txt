new_game(easy)	
new_game(easy)
--Start two easy games and see if it provides an erro
