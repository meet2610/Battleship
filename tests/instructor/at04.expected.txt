  state 0 OK -> Start a new game
->new_game(easy)
  state 1 OK -> Fire Away!
     1  2  3  4
  A  _  _  _  _
  B  _  _  _  _
  C  _  _  _  _
  D  _  _  _  _
  Current Game: 1
  Shots: 0/8
  Bombs: 0/2
  Score: 0/3 (Total: 0/3)
  Ships: 0/2
    2x1: Not Sunk
    1x1: Not Sunk
->new_game(easy)
  state 2 Game already started -> Fire Away!
     1  2  3  4
  A  _  _  _  _
  B  _  _  _  _
  C  _  _  _  _
  D  _  _  _  _
  Current Game: 1
  Shots: 0/8
  Bombs: 0/2
  Score: 0/3 (Total: 0/3)
  Ships: 0/2
    2x1: Not Sunk
    1x1: Not Sunk
